assets/app/styles/_extensions.less

```diff
@@ -0,0 +1,9 @@
+.pod-template div[hawtio-extension] {
+  position: relative;
+}
+
+.pod-template .connect-link {
+  position: absolute;
+  right: -5px;
+  top: -66px;
```

Comment:
Agreed, ideally the container-links extension would perhaps be before/after the 'name' in the template, then I can just right-justify it via 'right' and it'll be basically in the same spot.  I recall though being asked to put the extension in the template where it currently is, so it doesn't give me much choice but to work around it via css positioning.